ffi/diplomat/Cargo.toml

```
@@ -54,9 +54,9 @@ serde = [
     "icu_provider/serde",
     "icu_segmenter/serde",
 ]
-buffer_provider = ["serde", "icu_testdata/buffer"]
+buffer_provider = ["serde", "icu_provider_adapters/serde"]
```

You can enable features on optional dependencies without pulling them in with the `?` syntax:

```suggestion
buffer_provider = ["serde", "icu_provider_adapters/serde", "icu_testdata?/buffer"]
```

This will enable `icu_testdata/buffer` when `provider_test` and `buffer_provider` are enabled, but not when only one of them is.

---

Thanks